apps/pollbook/backend/src/networking.ts

```diff
@@ -102,8 +102,11 @@ export function fetchEventsFromConnectedPollbooks({
               return;
             }
             if (
-              !election ||
-              currentPollbookService.electionId !== election.id
+              !myMachineInformation?.electionBallotHash ||
+              myMachineInformation.electionBallotHash !==
+                currentPollbookService.electionBallotHash ||
```

Comment:
if `currentPollBookService.electionBallotHash` is guaranteed to not be undefined can this simplify down to 

```
 myMachineInformation?.electionBallotHash !==
                currentPollbookService.electionBallotHash || ...
```

Reply:
never mind i guess it's possible that both my machine and the peer machine are unconfigured and therefore have no election ballot hash?

Reply:
yeah I want to make sure that the current machine is configured